ansible/group_vars/masters.yml

```diff
@@ -2,7 +2,7 @@
 ####
 # Role - keepalived
 ####
-# keepalived_vip: ''
+keepalived_vip: ''
 # keepalived_interface: "{{ ansible_default_ipv4['interface'] }}"
```

Comment:
I think this section should be removed from the `masters.yml` file, it's already in the `all.yml` - feel it causes confusion. All nodes in the cluster need this IP, anyway.

Reply:
Done